services/cosi/cosi.go

```diff
@@ -41,24 +40,30 @@ type SignatureRequest struct {
 
 // SignatureResponse is what the Cosi service will reply to clients.
 type SignatureResponse struct {
-	Sum       []byte
+	Hash      []byte
 	Signature []byte
+	Aggregate abstract.Point
```

Comment:
Why sending back the aggregate public key ? The client is supposed to know the aggregate key already since he is the one giving the Roster to the service. And it's strange that a service gives back the key under which to verify the signature, it defeats the whole purpose of the signature ... ?